docs/tutorials/how-to-create-a-template.md

```diff
@@ -0,0 +1,75 @@
+## How to create and setup a Template
```

Comment:
Ok, so remember how you're adding a NEW **sub-section** to Tools (`/docs/tools/generator`)?  Thus, you will also be adding a `_section.md` to that as  **sub-section** similar to what we see here...

<img width="500" alt="section markdown files in our docs" src="https://user-images.githubusercontent.com/19964402/173472815-cd044d90-cb7e-46d7-8088-f51137335a27.png">

For **Generator Tool** _tutorials_, you should see a content outline and URL patterns such as...

- `docs/tutorials/generator-tool/installation-guide`
- `docs/tutorials/generator-tool/how-to-create-a-template`
- etc.

Reply:
P.S. This tutorial you're making is looking great!! I know your work is going to help our community a lot, thank you so much for your contributions 🎉✨✨✨✨

Reply:
> > For Generator Tool tutorials, you should see a content outline and URL patterns such as...
> 
> 
> 
> do you mean that tutorials for generator will not be in "tools" bucket, next to other generator-related docs but in "tutorials"?

@derberg You got it